DataFormats/SiStripCluster/src/classes_def.xml

```diff
@@ -24,8 +24,8 @@
  <class name="edm::Wrapper<edmNew::DetSetVector<edm::Ref<edmNew::DetSetVector<SiStripCluster>,SiStripCluster,edmNew::DetSetVector<SiStripCluster>::FindForDetSetVector> > >" />
 
 
- <class name="SiStripApproximateCluster" ClassVersion="3">
-  <version ClassVersion="3" checksum="2041370183"/>
+ <class name="SiStripApproximateCluster" ClassVersion="9">
+  <version ClassVersion="9" checksum="2854791577"/>
```

Comment:
why did it jump from 3 -> 9?

Reply:
I have updated this file to be version '4' now.